src/components/ProjectSettingsTabs/VariablesTab/getFormSchema.ts

```diff
@@ -37,6 +37,10 @@ export const getFormSchema = (
               type: "boolean" as "boolean",
               title: "Private",
             },
+            isAdminOnly: {
```

Comment:
I have no idea how to fix this but: I made "my last variable" both private and admin-only (which is visible in the DB: 
```
{
	"_id" : "evergreen",
	"vars" : {
		"my new variable" : "value!",
		"another variable!" : "so valuable",
		"my last variable" : "the most valuable"
	},
	"private_vars" : {
		"another variable!" : true,
		"my last variable" : true
	},
	"admin_only_vars" : {
		"my last variable" : true,
		"my new variable" : true
	},
	"restricted_vars" : null
}
```
but not in the UI:
![image](https://user-images.githubusercontent.com/26798134/155558190-c1fe5f8f-66b5-4ac4-aa72-ee02ceaa140c.png)

Reply:
fixed!

Reply:
I pulled your branch and the evergreen repo, and I'm still seeing this for non-repo variables:
![image](https://user-images.githubusercontent.com/26798134/155775329-e1c8df3f-274f-42a7-be20-7abd9d3b4185.png)

I actually also think something is very broken and I'm assuming this is your changes since the DB thing I posted above doesn't have this problem, but I'm not sure: private vars are now having their value saving as REDACTED even though that shouldn't be saved. cc @malikchaya2  @sophstad 
```MongoDB Enterprise rs0:PRIMARY> db.project_vars.findOne({"_id":"evergreen"})
{
	"_id" : "evergreen",
	"vars" : {
		"another variable!" : "{REDACTED}",
		"my last variable" : "{REDACTED}",
		"my dual variable" : "yee"
	},
	"private_vars" : {
		"another variable!" : true,
		"my last variable" : true,
		"my dual variable" : true
	},
	"admin_only_vars" : {
		"my dual variable" : true
	},
	"restricted_vars" : null
}
```

Reply:
@bynn: sorry for the scare. This actually does look like an existing bug. @malikchaya2 could you ticket? I think this is a backend issue but I'm not sure.

Reply:
Based on a quick look I actually wouldn’t be surprised if this were originating on the front end, I’ll take a look next week!

Reply:
EVG-16419